### DISCUSSION TITLE
People Webpart content not loading when having many webparts in one page

### DISCUSSION THREAD

**Discussion by Heavm00** [microsoft-search/pnp-modern-search#3638] [Q&A]
I have different pages where I use up to 6 PnPSearch results webparts where people from the organisation are shown. Randomly the webparts do not show any content. When I want to switch to edit mode, I'm facing the following problem:

![image](https://github.com/microsoft-search/pnp-modern-search/assets/61208008/b93f12d8-2168-4e3f-9d61-c62627d009fb)

Does anybody know about that problem?

**Comment by wobba** [comment]
If it's randomly not being able to load SPFx assets from your app catalgot, then that is a SPFx platform issue, not a web part issue per say. That said, you could benefit from enabling the CDN capability https://learn.microsoft.com/en-us/sharepoint/dev/spfx/web-parts/get-started/hosting-webpart-from-office-365-cdn

**Comment by Heavm00** [comment]
Hi @wobba, thanks for your response. Enabling the CDN does not change anything. 
Could there be a problem, because the app has been installed for each site (and not tenant-wide). 

Do you have any other ideas why it's not working in my case?

Thanks!

**Reply by wobba** [reply]
Right.. so it's site app catalog installed. Having 6 search parts on a page is not recommended btw, but they should still not fail to load.  I'll see if I can repro and figure it out - but cannot promise any timelines.

**Comment by Heavm00** [comment]
Actually, enabling the CDN had more bad effekts - the search result page now looks like:

![image](https://github.com/microsoft-search/pnp-modern-search/assets/61208008/3ee99a04-c912-464f-80c5-8c5851a04868)

There are also different scenarios: 

- when I'm logged in from my company network nothing is shown correctly (since enabling CDN)
- when I'm looged in from my private PC at home (no VPN): webparts loading randomy correct ord shoing the above error messages.

**Reply by wobba** [reply]
Can you try to delete the app and re-add/upload it to the site? This will not impact pages with the web part. And you can disable CDN if you want - but in general it's best to leave it on for loading performance of SPFx + images on pages.
